invenio_i18n/ext.py

```diff
@@ -137,8 +137,8 @@ def iter_languages(self):
 
         yield (default_lang, default_title)
 
-        for l, title in current_app.config.get('I18N_LANGUAGES', []):
-            yield l, title
+        for language, title in current_app.config.get('I18N_LANGUAGES', []):
```

Comment:
OCD: sometimes `language`, sometimes `lan`, sometimes `lang`. Would be nice, as long as scopes allow to have always the same naming

Reply:
Addressed 👍